coreapi/sfdc/src/main/java/com/jalasoft/sfdc/ui/pages/FormBookBaseLight.java

```diff
@@ -0,0 +1,38 @@
+package com.jalasoft.sfdc.ui.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class FormBookBaseLight extends FormBoookPage {
+
+    @FindBy(xpath = "//*[@id=\"7:1570;a\"]")
+    WebElement nameB;
+    @FindBy(xpath = "//*[@id=\"43:1570;a\"]")
+    WebElement descript;
+//    @FindBy(xpath ="//*[@id=\"bottomButtonRow\"]/input[1]" )
+//    WebElement saveBtn;
+
+    public  FormBoookPage clickNewBook(){
+        return new FormBookBaseLight();    }
+
+
+    private void setNameBook(String namebook){
```

Comment:
delete the methods and use the next method:
@override
public PriceBooksDetailPage addNewPriceBooks(String priceBookName, String description) {
driverTools.setInputField(priceBooksName, priceBookName);
driverTools.setInputField(description, description);
return new PriceBooksDetailPageClassic();
}
your methods have to be more descriptive